src/realm/object-store/c_api/realm.cpp

```diff
@@ -173,6 +179,25 @@ RLM_API realm_callback_token_t* realm_add_realm_changed_callback(realm_t* realm,
         realm, CBindingContext::get(*realm).realm_changed_callbacks().add(std::move(func)));
 }
 
+RLM_API realm_refresh_callback_token_t* realm_add_realm_refresh_callback(realm_t* realm,
+                                                                         realm_on_realm_refresh_func_t callback,
+                                                                         realm_userdata_t userdata,
+                                                                         realm_free_userdata_func_t userdata_free)
+{
+    util::UniqueFunction<void()> func = [callback, userdata = UserdataPtr{userdata, userdata_free}]() {
+        callback(userdata.get());
+    };
+    const auto& current_snap_version = (*realm)->current_transaction_snapshot_version();
+    const auto& current_realm_version = (*realm)->current_transaction_version();
```

Comment:
There doesn't appear to be any reason to get this?

Reply:
no, actually it was just for computing a check latest != current (but it cannot be done within the current write transaction, so I guess it is just stale code)